package.json

```diff
@@ -21,25 +21,25 @@
   },
   "homepage": "https://github.com/fastify/fastify-accepts-serializer#readme",
   "devDependencies": {
-    "coveralls": "^2.13.1",
-    "fastify": "^0.26.0",
-    "msgpack5": "^3.5.0",
+    "coveralls": "^3.0.2",
+    "fastify": "^1.13.0",
+    "msgpack5": "^4.2.1",
     "pre-commit": "^1.2.2",
-    "protobufjs": "^6.8.0",
-    "shot": "^3.4.2",
-    "snazzy": "^7.0.0",
-    "standard": "^10.0.2",
-    "tap": "^10.7.1",
+    "protobufjs": "^6.8.8",
+    "shot": "^4.0.7",
+    "snazzy": "^8.0.0",
+    "standard": "^12.0.1",
+    "tap": "^12.1.0",
     "yamljs": "^0.3.0",
-    "mocha": "^3.5.0",
-    "nyc": "^11.1.0"
+    "mocha": "^5.2.0",
+    "nyc": "^13.1.0"
   },
   "peerDependencies": {
-    "fastify": ">=0.26.0"
+    "fastify": ">=1.13.0"
   },
   "dependencies": {
-    "boom": "^5.2.0",
-    "fastify-accepts": ">=0.2.0",
-    "fastify-plugin": "^0.1.0"
+    "boom": "^7.2.2",
```

Comment:
do we still need boom? I would prefer if we move this to http-errors instead.

Reply:
Not really. I saved it for the real next branch but lets drop it now.